module2/model/src/main/java/com/epam/esm/model/Tag.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.esm.model;
+
+import java.util.Objects;
+
+public class Tag {
+    private Long id;
+    private String name;
+
+    public Tag() {
```

Comment:
Make sure that you use all constructors, that you have written

Reply:
empty constructor is used by spring for tag bean initiliazation

Reply:
@pilinoga, is Tag managed by spring? Is it bean?